packages/cxl-jwplayer/src/plugins/cxlJWChapters.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable */
```

Comment:
Let's convert to better file names immediately: https://developers.google.com/style/filenames `cxl-jw-chapters.js` etc

Reply:
Is this PR even valid vs #69?